crates/networking/p2p/net.rs

```diff
@@ -49,8 +49,8 @@ pub async fn start_network(
     signer: SigningKey,
     storage: Store,
 ) {
-    info!("Starting discovery service at {udp_addr}");
-    info!("Listening for requests at {tcp_addr}");
+    debug!("Starting discovery service at {udp_addr}");
+    debug!("Listening for requests at {tcp_addr}");
```

Comment:
I'd rather leave these two lines as `info!` as they appear only once per run in the console.